src/main/java/com/sarahhand/fractals/model/colorscheme/mandelbrotset/ExternalDistanceEstimateColorScheme.java

```diff
@@ -0,0 +1,91 @@
+package com.sarahhand.fractals.model.colorscheme.mandelbrotset;
+
+import java.awt.Color;
+
+import com.sarahhand.fractals.model.ColorPalette;
+import com.sarahhand.fractals.model.ComplexNumber;
+import com.sarahhand.fractals.model.FractalConfig;
+import com.sarahhand.fractals.model.MandelbrotConfig;
+import com.sarahhand.fractals.model.MandelbrotPointData;
+import com.sarahhand.fractals.model.PointData;
+import com.sarahhand.fractals.model.colorscheme.ColorScheme;
+
+public class ExternalDistanceEstimateColorScheme implements ColorScheme{
+
+	@Override
+	public String getName(){
+		return "Exterior Distance";
+	}
+
+	@Override
+	public Color getColor(PointData data, FractalConfig config){
+		
+		if(!(data instanceof MandelbrotPointData)){
+			
+			throw new IllegalArgumentException("data must be instance of MandelbrotPointData");
+		}
+		
+		if(!(config instanceof MandelbrotConfig)){
+			
+			throw new IllegalArgumentException("config must be instance of MandelbrotConfig");
+		}
+		
+		MandelbrotPointData castData = (MandelbrotPointData)data;
+		MandelbrotConfig castConfig = (MandelbrotConfig)config;
+		
+		ComplexNumber pnc = castData.getZValues().get(castData.getEscapeTime());
+		double absPnc = Math.sqrt(pnc.x*pnc.x+pnc.y*pnc.y);
+		
+		ComplexNumber one = new ComplexNumber(1, 0);
```

Comment:
Can you make these two values constants inside ComplexNumber so that they don't have to be recreated each time this method is called?